### PR TITLE
I now harness power from the spirits.

### PR DESCRIPTION
**Pull request by Iamgoofball**
Requires #19227

### CODE REVIEW

File: code/modules/power/tesla/energy_ball.dm [hunk 1]
```diff
@@ -66,8 +66,12 @@ var/list/blacklisted_tesla_types = typecacheof(list(/obj/machinery/atmospherics,
...
 		pixel_y = -32
 		for (var/ball in orbiting_balls)
-			var/range = rand(1, Clamp(orbiting_balls.len, 3, 7))
-			tesla_zap(ball, range, TESLA_MINI_POWER/7*range)
+			if(orbiters.len >= 1)
+				var/range = rand(1, Clamp(orbiting_balls.len, 3, 7)) * (orbiters.len / 2)
+				tesla_zap(ball, range, TESLA_MINI_POWER/7*range)
+			else
+				var/range = rand(1, Clamp(orbiting_balls.len, 3, 7))
+				tesla_zap(ball, range, TESLA_MINI_POWER/7*range)
```
Comment (MrStonedOne) [inline code/modules/power/tesla/energy_ball.dm]: I should point out that the tesla's orbiting balls would also be on this list

Comment (Iamgoofball) [inline code/modules/power/tesla/energy_ball.dm]: shit

Comment (Jalleo) [inline code/modules/power/tesla/energy_ball.dm]: Tesla shooting itself nice

**Comment by optimumtact** [comment]
Update your PR OP to include more description of what this is please.

**Comment by optimumtact** [comment]
Will reopen when unblocked

**Pull request closed without merging**